src/test/java/teammates/test/cases/testdriver/TestngTest.java

```diff
@@ -0,0 +1,75 @@
+package teammates.test.cases.testdriver;
+
+import static org.testng.AssertJUnit.assertTrue;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.testng.annotations.Test;
+
+import teammates.common.util.FileHelper;
+import teammates.test.cases.BaseTestCase;
+
+public class TestngTest extends BaseTestCase {
+
+    @Test
+    public void checkTestsInTestng() throws FileNotFoundException {
+        HashMap<String, String> tests = new HashMap<String, String>(); // <class name, package name>     
+        String testngXmlAsString = FileHelper.readFile("./src/test/testng.xml");
+        
+        addFilesToTestsRecursively(tests, "./src/test/java/teammates/test/cases", true, "", 
+                                       testngXmlAsString); // BaseComponentTestCase, BaseTestCase (files in current directory) excluded
+        
+        tests = excludeFile(tests, "BaseUiTestCase");
+        tests = excludeFile(tests, "FeedbackQuestionUiTest");
+        tests = excludeFile(tests, "GodModeTest");
+
+        verifyTestngContainsTests(tests, testngXmlAsString);
+    }
+    
+    private HashMap<String, String> excludeFile(HashMap<String, String> tests, String file) {
```

Comment:
Now this method has no difference from `tests.remove(...)`. You can remove the `file` parameter and remove the three unwanted tests in the method body, and name the method like how you did previously.